OloEngine/src/OloEngine/Renderer/Commands/CommandBucket.cpp

```
@@ -278,29 +356,19 @@ namespace OloEngine
-                instancedCmd->transforms[instancedCmd->instanceCount++] = sourceCmd->transform;
-                return true;
-            }
-
+            // Note: With POD commands, transforms are stored in FrameDataBuffer
+            // Dynamic instance batching would require allocating new FrameDataBuffer space
+            // For now, return false to skip dynamic batching (it's handled at command creation time)
+            // TODO: Implement proper FrameDataBuffer-based instance batching
+            OLO_CORE_WARN("CommandBucket: Dynamic instancing with POD commands not yet fully implemented");
             return false;
         }
         else if (targetType == CommandType::DrawMeshInstanced && sourceType == CommandType::DrawMesh)
```

Clang (14.0.6)

[warning] do not use 'else' after 'return'

(readability-else-after-return)